packages/common/src/api/tan-query/jupiter/useSwapTokens.ts

```diff
@@ -104,7 +108,7 @@ export const useSwapTokens = () => {
           amountUi,
           slippageBps,
           swapMode: 'ExactIn',
-          onlyDirectRoutes: true
+          onlyDirectRoutes: false
```

Comment:
needed to enable non-USDC paired swaps

Reply:
wait, is this code shared in other parts of the app like withdraws etc? if so maybe we should keep those settings for that, and then make a new hook for artist coin swaps?

sry if that's already the case tho

Reply:
yea this is only the buy/sell flow

Reply:
I think in the future if we have more call sites we'll want to parameterize this hook and have the callers be able to pass in custom options. Good for now though!